examples/htlc-pyteal-ts/scripts/withdraw/common.ts

```diff
@@ -0,0 +1,30 @@
+import { executeTransaction } from "@algorand-builder/algob";
+import { Account, AlgobDeployer, ExecParams } from "@algorand-builder/algob/src/types";
```

Comment:
can we export these from `algob` ? then we can directly import them in above line

Reply:
ok

Reply:
Please don't change this.
https://github.com/scale-it/algorand-builder/pull/209#discussion_r567873745